bindings/python/examples/simple_transaction/request_funds.py

```diff
@@ -12,6 +13,13 @@
 
 account = wallet.get_account('Alice')
 
+# Check and use password
+if 'STRONGHOLD_PASSWORD' not in os.environ:
+    print(".env STRONGHOLD_PASSWORD is undefined, see .env.example")
+    sys.exit(1)
+
+wallet.set_stronghold_password(os.environ["STRONGHOLD_PASSWORD"])
+
```

Comment:
```suggestion
```
Stronghold password is only required for address generation/signing :trollface:

Reply:
Done